test/lib/Resource.js

```diff
@@ -75,6 +75,15 @@ test("Resource: getBuffer with throwing an error", (t) => {
 	});
 });
 
+test("Resource: getPath / getName", (t) => {
+	const resource = new Resource({
```

Comment:
could you add a test for executing `setPath` after the resource is created and check if name is updated, too?

Reply:
Thanks! Forgot about that.